app/audit/AuditHelper.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit
+
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import models.AuditModel
+import play.api.mvc.Request
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.AuditExtensions
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.play.audit.model.DataEvent
+
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class AuditHelper @Inject()(appConfig: AppConfig, auditConnector: AuditConnector) {
+  def audit(auditModel: AuditModel, path: Option[String] = None)(implicit hc: HeaderCarrier, request: Request[_], ec: ExecutionContext): Unit = {
```

Comment:
Could you explain why we're passing in the path argument to this function, when we're only using the request path for the DataEvent tags field?

Reply:
That would be legacy from copying the functionality from another service, I don't think we use it so I will remove it.

Reply:
OK, but at the moment are we using the optional path at all? Could we not remove this argument and just have the path from the request